src/ViewItem.js

```diff
@@ -179,6 +189,29 @@ class ViewItem extends React.Component {
     }
   }
 
+  async componentDidUpdate(prevProps) {
+    if (prevProps.resources.items && this.props.resources.items.records[0] !== prevProps.resources.items.records[0]) {
+      const { lastCheckIn } = this.props.resources.items.records[0];
+
+      if (lastCheckIn) {
+        const { staffMemberId, servicePointId } = lastCheckIn;
+
+        const [staffMembers, servicePoints] = await Promise.all(
```

Comment:
```
        const [staffMembers, servicePoints] = await Promise.all(
          [
            this.fetchEntityById(staffMemberId, 'borrowers'),
            this.fetchEntityById(servicePointId, 'servicePoints'),
          ]
        );
```

It seems, it will be better to format:

```
        const [staffMembers, servicePoints] = await Promise.all([
          this.fetchEntityById(staffMemberId, 'borrowers'),
          this.fetchEntityById(servicePointId, 'servicePoints'),
        ]);
```